### ISSUE TITLE
USD/Day WAY Off?

### ISSUE THREAD

**Issue by crazycoder4life** [Sniffdog/Sniffdogminer#37]
So I just started using Sniffdog, and the USD/Day evaluation on my single 1080ti is 18.973 for Skein. For me 1070ti x3 rig, it is 258USD for blake2s (although it has been bouncing between 40-50?) .. NONE of these values can be right, but the BTC value it is pulling from the coinbase API seems to be right. Any ideas?

**Comment by frederikvanduuren** [comment]
Hi,
Indeed...

I have the same low mining drop lately :( from $50/day to $15/day!

any ideas what changed after the last upgrade?
fred

**Comment by Sniffdog** [comment]
yeah the basic understanding with pool API's is that these "pools do move" hash around through price changing... 

We as miners just need to let the cycle run or change our strategy to pick only a few algo's or solo mine one algo..

If you want more answers you will have email the pool operator

**Comment by crazycoder4life** [comment]
It actually hasn’t been too far off here lately! It’s back down to saying about 30 USD a day which is what I actually get. Way better than solo mining or mining a few IMO. I think it was something with the x17 alg specifically, because the pool it was using put up a message about it. Thanks for this piece of art, and thanks for getting back with me. :)

**Comment by Sniffdog** [comment]
oh yeah verge was pumping so many new miners... 

Also, remember nicehash hack a while back??? well the "solving diff" dropped causing more coins to be mined by other pools... now the diff is almost back to normal... why also you saw increase then decrease

pumps and dumps and hacks is always with mining... but Sniffdog will pick it out for yah

**Comment by dariofb** [comment]
Yup, started one week ago with sniffdog and i have to say its an awesome miner. Best so far.

 But i agree with the feeling thats something wrong with the USD/Day earnings that sniffdog shows and the actual pool payouts (less than de 50%). 

1 Nvidia 1080 Ti showing 10-30 USD/Day of earnings vs 7/8 USD/Day pool payouts.

Im sure its not sniffdogs fault but i would like to know "whats going on" to understand the difference...

(Only using ahashpool now)

**Comment by frederikvanduuren** [comment]
the USD/Day earnings  is not an average, but just the current peak earnings for that time.
from the moment you see these high numbers, it will drop within the next few seconds/minutes because most miners mine asap the high rewards...
run your script every x seconds & see what happens with the top 3... a battle ;)

**Comment by crazycoder4life** [comment]
Do you all find it better to use a higher than default interval for the
watchdog? I feel like it changes so often that the miner doesn’t have time
to “ramp up” so to speak before it’s trykng to switch again.

**Comment by frederikvanduuren** [comment]
difficult question....

i tend to lower the frequency with faster rampup algos...

to be benchmarked ;)

**Comment by crazycoder4life** [comment]
Can you elaborate on that? Would be highly appreciated

**Comment by frederikvanduuren** [comment]
ok here we go with the logic...
suppose you rescan every 90 seconds...
but some algos/pools take sometimes 45 to 60 seconds to start mining...
so instead to use higher intervals like 120 or 240
i tend to lower the interval to 47 seconds and skip the higher startup algo/pools!

would love to benchmark each one of them seperatly and only run the faster algos with the higher earnings :)

**Comment by Sniffdog** [comment]
Yes Fred your assumption are correct...

crazycroc... the default interval is just for benching... after the benching is complete try to experiment with different interval times that suits you.

Fred like 47sec interval

ahashpool like interval 300

i like 240

It's really up to your preference... I will elaborate on my setting...

I have been studying ccminer and noticed that a 240 interval ramps up my 1080ti's to almost half hashing power at the algo(a)(where the miner is hashing at). When Sniffdog rechecks at the next 240 interval... and nothing changes... then the miner will hash at full hash on that algo(a).

If it changes when Sniffdog rechecks at the 240 interval and moves to a different algo(b).... the miner still has work to complete on the algo(a) as well as begin on algo(b) so it will take time for algo(a) to complete before you can run at full hash on algo(b)...

So I'm trying to spread out the work load and trying to keep the (%shares) up on the algo's

**Comment by frederikvanduuren** [comment]
i dont believe its a preference,
i just get higher mining earnings...

will try 240 for once ;)

my x6 gtx1070 now produces on average $26 (but i had $50 2 weeks ago & just $14 last week)

too many params to work with...

**Comment by Sniffdog** [comment]
gotta remember there is pump and dumps happening in the market as well to

**Comment by frederikvanduuren** [comment]
like i said: too many params to work with ;)

need an AI brain to figure these polys
